src/utils/utils.ts

```diff
@@ -179,7 +179,7 @@ export function getProperty (value: { [key: string]: any}, key: string): any {
   }
   
   const keys: string[] = key.split('.');
-  let result: any = value[keys.shift()];
+  let result: any = value[keys.shift()!];
```

Comment:
Right, just tested in a new project, seems to be a problem on my side.
Sorry for the trouble.